pretext/project/__init__.py

```diff
@@ -906,7 +908,7 @@ class Project(pxml.BaseXmlModel, tag="project", search_mode=SearchMode.UNORDERED
     """
 
     model_config = ConfigDict(extra="forbid")
-    ptx_version: t.Literal["2"] = pxml.attr(name="ptx-version")
+    ptx_version: t.Literal["2"] = pxml.attr(name="ptx-version", default="2")
```

Comment:
Won't this make any manifest without a `ptx-version` attribute be recognized as version 2?

Reply:
I think we have different logic when running `Project.parse()`. The goal here is to let `Project()` work without raising an error.